packages/experimental/Badge/src/PresenceBadge/PresenceBadge.types.ts

```diff
@@ -0,0 +1,18 @@
+import { BadgeProps } from '../';
+
+export const presenceBadgeName = 'PresenceBadge';
+export type Presence = 'DND' | 'busy' | 'unknown' | 'blocked' | 'OOF' | 'away' | 'available' | 'offline';
+
+export interface PresenceBadgeProps extends BadgeProps {
+  presence?: Presence;
+  oof?: boolean;
```

Comment:
I thought the spec had the prop as outOfOffice, and not oof? I think spelling it out would be better as it's not obvious what oof stands for

Reply:
+1 to not using acronyms.